lib/index.js

```diff
@@ -1,18 +1,29 @@
-const { compress: _compress, decompress: _decompress } = require('bindings')('zstd');
+const zstd = require('bindings')('zstd');
```

Comment:
Aside from the `bindings` package discussion, you probably want

```suggestion
'use strict';
const zstd = require('bindings')('zstd');
```

(because, you know, it's not the 90s anymore 😉)

Reply:
eslint complains about adding `use strict` because it says that 'use strict' isn't necessary for modules.

Reply:
It erroneously thinks this is a module because the sourceType isn't set I think:

```json
"parserOptions": {
    "sourceType": "commonjs",
```